ts/RailroadMap.ts

```diff
@@ -115,6 +115,9 @@ const DEFAULT_LAYER_VISIBILITY: MapLayerVisibility = {
     turntables: true,
 };
 
+// this is just a guess and could be determined with binary search given enough old savegames
+const ROTATION_FIX_SAVE_GAME_VERSION = 231117;
```

Comment:
I have verified that the version number corresponds with the map update.

```suggestion
const INVERT_BEFORE_SAVE_GAME_VERSION = 231117;
```